src/style.css

```diff
@@ -0,0 +1,239 @@
+/* PRIMARY */
+/* NAV */
+/* MAIN */
+/* FORM */
+/* FOOTER */
```

Comment:
Do you really need this comment? If so, make it descriptive, so it will be very easy for you to understand it in future, or even your fellow programmers will want to understand your codes, otherwise I'd recommend you to remove them from your codes

Reply:
Here we just try to make a Table of Contents for CSS file, like is here. https://speckyboy.com/good-bad-css-practices/